docs/other/preferences.rst

```diff
@@ -12,9 +12,9 @@ The Preferences object provides an easy way to manage internal AE preferences, s
 
 Preferences are identified by section and key within the file, and each key name is associated with a value.
 
-In the preferences file, section names are enclosed in brackets and quotation marks, and key names are listing in quotation marks below the sectionname. All values are strings.
```

Comment:
Is the 'sectionname' referred here a property name? If so I'll revert this part and the one in settings.rst :bow:

Reply:
It's the name of the section in the preferences. 
> Preferences are identified by section and key within the file, and each key name is associated with a value.
[Source](https://github.com/docsforadobe/after-effects-scripting-guide/blob/72af50d54349509daf08c179a2d65c6eea2843e6/docs/other/preferences.rst?plain=1#L13)